main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"sync"
+)
+
+// Common wake on lan ports
+var ports = []int{7, 9}
+
+func constructMagicPacket(macAddress net.HardwareAddr) []byte {
+	magic := append([]byte{0xff, 0xff, 0xff, 0xff, 0xff, 0xff})
```

Comment:
Why do you use `append` here and not `magic := []byte{0xff, 0xff, 0xff, 0xff, 0xff, 0xff}`?

Reply:
this was a missed clean up